code/tools/run_governed_response_composer.py

```
@@ -0,0 +1,620 @@
+        if result.returncode != 0:
+            error_msg = result.stderr.strip() if result.stderr else "Unknown error"
+            raise RuntimeError(f"Claude CLI error: {error_msg}")
+        
+        # Parse JSON output
+        try:
+            response_data = json.loads(result.stdout)
+            # Extract text from Claude API format
+            if "content" in response_data and len(response_data["content"]) > 0:
+                return response_data["content"][0]["text"]
+            else:
+                raise ValueError("Unexpected response format from Claude CLI")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)